example/image_classification_mobilenet/lib/helper/image_classification_helper.dart

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'dart:developer';
+import 'dart:io';
+import 'dart:isolate';
+
+import 'package:camera/camera.dart';
+import 'package:flutter/services.dart';
+import 'package:image/image.dart';
+import 'package:tflite_flutter/tflite_flutter.dart';
+
+import 'isolate_inference.dart';
+
+class ImageClassificationHelper {
+  static const modelPath = 'assets/models/mobilenet_quant.tflite';
```

Comment:
I believe we can use the model that has metadata for labels rather than the labels.txt file. We should also try putting together a script to download the models (use the same model from MediaPipe and one of the existing Flutter samples already handles downloading)

Reply:
I take back the part about labels - just reviewed how that's working in the current example. Might still be the case, but we can adjust that later.